pkgs/by-name/te/testkube/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "testkube";
+  version = "2.1.56";
+
+  src = fetchFromGitHub {
+    owner = "kubeshop";
+    repo = "testkube";
+    rev = "v${version}";
+    sha256 = "sha256-P+A9lUMzQ3M0SEVZBMDSMj8S0uCsEhadv5vDRxbQORA=";
```

Comment:
```suggestion
    hash = "sha256-P+A9lUMzQ3M0SEVZBMDSMj8S0uCsEhadv5vDRxbQORA=";
```

Reply:
fixed in e25f98f28ddc2fc7ced25ce5bcde8e84293168a2